### ISSUE TITLE
How do you pay living costs during funemployment?

### ISSUE THREAD

**Issue by lukechilds** [sindresorhus/ama#383]
From your [blog](https://blog.sindresorhus.com/about-sindre-sorhus-42786d2e191b):

> Currently, I’m funemployed traveling around Southeast Asia doing open source. Say hi if you’re around!

Did you save up funds beforehand? Stop every now and then to do a bit of freelance work?

I'm currently considering doing something similar, any advice would be appreciated 😄

**Comment by sindresorhus** [comment]
Yes, I saved up some money, which I'm still living off. Money last longer in Southeast Asia than in Norway, and the UK too I assume. I plan on doing some freelance work when I need money again. Would definitely recommend doing it! Don't overthink it, just do it. Happy to answer any specific questions you might be having.

**Comment by lukechilds** [comment]
Thanks, I definitely think I will, it's more a matter of when than if as I'm kind of tied up in some big projects at work at the moment.

What does an average day involve? Would you say you spend the majority of your time coding in a hotel room or being outside exploring?

**Comment by sindresorhus** [comment]
Depends on whether I'm traveling. When traveling, like I'm doing at the moment (currently in Vietnam), I explore outside, especially trying out the local food, and I try to force myself to meet up with people. In the middle of the day I usually sit in coffee shops as it's very hot outside. [When not traveling](https://github.com/sindresorhus/ama/issues/167#issuecomment-141025071), I can be spotted in various coffee shops around Bangkok, hacking on some open source projects.

**Comment by lukechilds** [comment]
Awesome, thanks 👍 

I may come back here if I have some more specific questions.

**Comment by lukechilds** [comment]
I've been doing a bit more research on this and I think I've decided on Chiang Mai. They have some pretty cool coworking spaces which I think would be good as I was kind of worried about getting a bit lonely.

Is there any reason you prefer coffee shops over coworking spaces? Also any reason you prefer Bangkok in particular?

**Comment by sindresorhus** [comment]
Chiang Mai is a great choice. Fast internet, many coworking spaces, cheap living expenses, and nice nature. I personally prefer coffee shops as they have good coffee and I can get some variation in scenery by trying out different coffee shops. And there's something about the background noise of people chatting in coffee shops that makes me more productive. Bangkok is an awesome city, enormous selection of food options, always some events happening, lots of great cinemas, coffee shops, etc, but it's also huge, noisy and expensive compared to Chiang Mai. I think both are worth trying out. I want to live in Chiang Mai at some point too.

**Comment by jfmengels** [comment]
> Money last longer in Southeast Asia than in Norway

I'm curious as to how much it costs. What would you guess to be your average monthly bill? I'm keeping in mind that you don't buy a lot of stuff.

**Comment by sindresorhus** [comment]
Guesstimate:

- Rent: $500 (This apartment is as central as it could get. You can get something nice for half that if you're willing to travel 15-20 min to the city core of Bangkok. Apartments are also much cheaper outside the city.)
- Food: $500 (I eat out every day because food is so cheap here)
- Transport: $80 (Taxis)
- Fun: $100 (Cinema, partying, etc)

**Comment by jfmengels** [comment]
Thanks! :)

**Comment by lukechilds** [comment]
Hey man, I arrived in Bangkok last night! 🎉 

I'm doing some traveling with a friend for a month or so first and then I'm gonna crack on with some work. We've got kayaking booked near Phuket in a few days so we need to get down there and I'd rather store my laptop and a few other tech bits somewhere secure whilst traveling.

Have you used any secure storage in Bangkok or anywhere else in Thailand? Heard any recommendations of somewhere good? I found [bangkokselfstorage.com](http://bangkokselfstorage.com/) but the reviews don't fill me with confidence. Any info would be a great help.
